workflows/ort-analyze.yml

```diff
@@ -0,0 +1,79 @@
+# For documentation about OSS Review Toolkit see https://github.com/oss-review-toolkit/ort
+# Based on: https://github.com/porscheofficial/oss-review-toolkit-action/blob/v1/action.yml

```

Comment:
Shall this comment be removed?

Reply:
Good point, the repository doesn't exist anymore. I removed it as proposed.